spyder/utils/introspection/utils.py

```diff
@@ -211,6 +211,30 @@ def get_keywords(lexer):
     return keywords
 
 
+def get_words_file(file_in):
```

Comment:
This means the logic you are using for getting the extension, needs to be an extra parameter as well.